### PR TITLE
Actual implementation of the microservice

### PR DESCRIPTION
**Pull request by izzyblues**

### CODE REVIEW

File: concepts/concordances_test.go [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+}
+
+func TestGetConcordancesFailsInvalidURL(t *testing.T) {
+	concordances := NewConcordances(&http.Client{}, ":#")
+	_, err := concordances.GetConcordances("tid_TestGetConcordancesFailsInvalidURL", uuid.NewV4().String())
+
+	assert.Error(t, err)
+}
+
+func TestGetConcordancesRequestFails(t *testing.T) {
```
Comment (galiri) [inline concepts/concordances_test.go]: This is identical to the one above TestGetConcordancesFailsInvalidURL

File: resources/internal_concordances_test.go [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+
+	req := httptest.NewRequest("GET", "/?ids=a-concorded-uuid", nil)
+	req.Header.Add("X-Request-Id", "tid_TestSearchByIDs")
+	w := httptest.NewRecorder()
+
+	identifiers := map[string][]concepts.Identifier{
+		"a-uuid": []concepts.Identifier{
+			{Authority: "authority", IdentifierValue: "a-concorded-uuid"},
+		},
+	}
```
Comment (galiri) [inline resources/internal_concordances_test.go]: question about this data- if it was real life would it include an Identifier for "a-uuid" as well?

File: health/healthchecks_test.go [hunk 3]
```diff
@@ -19,12 +22,54 @@ func TestHealthServiceHandler(t *testing.T) {
 
 	assert.Equal(t, http.StatusOK, w.Code)
 	assert.NotEmpty(t, w.Body)
+
+	dec := json.NewDecoder(w.Body)
+	var r fthealth.HealthResult
+	err := dec.Decode(&r)
+	assert.NoError(t, err)
+	assert.Len(t, r.Checks, 2)
```
Comment (galiri) [inline health/healthchecks_test.go]: shouldn't this also assert it's unhealthy?

File: resources/internal_concordances_test.go (hunk 2 above)
Comment (peteclark-ft) [reply resources/internal_concordances_test.go]: Yes that's correct - so there would likely be two `Identifier` records, one for the old UUID

File: concepts/concordances_test.go (hunk 1 above)
Comment (peteclark-ft) [reply concepts/concordances_test.go]: It's subtly different... Using a url of `:#` causes `url.Parse` to fail, which in turn causes the `http.NewRequest()` line to `err`. 

Using the inverse url, `#:` causes `client.Do(request)` line to fail due to an invalid protocol `err`.

**Pull request merged**